app/parse_txs.go

```diff
@@ -20,3 +23,66 @@ func separateTxs(txcfg client.TxConfig, rawTxs [][]byte) ([][]byte, []core.BlobT
 	}
 	return normalTxs, blobTxs
 }
+
+// filterStdTxs applies the provided antehandler to each transaction and removes
+// transactions that return an error. Panics are caught by the checkTxValidity
+// function used to apply the ante handler.
+func filterStdTxs(logger log.Logger, dec sdk.TxDecoder, ctx sdk.Context, handler sdk.AnteHandler, txs [][]byte) ([][]byte, sdk.Context) {
```

Comment:
I didn't add unit tests for these, since those tests are essentially identical to the `TestPrepareProposalFiltering`, in both what we test and structure of the test itself